pom.xml

```diff
@@ -64,7 +64,7 @@
 
   <properties>
     <!-- versions -->
-    <version.sonar>6.4</version.sonar>
+    <version.sonar>6.7</version.sonar>
```

Comment:
I think it would be better to create a separate ticket for this little change.

Reply:
`BuiltInQualityProfilesDefinition` was introduced in SQ API 6.6, so the other changes are based on this update. But I can make a separate commit for it so that we keep a trace in commit history